larray/core/axis.py

```diff
@@ -1067,12 +1067,17 @@ def union(self, other):
 
         Examples
         --------
-        >>> letters = Axis('letters=a,b')
-        >>> letters.union(Axis('letters=b,c'))
-        Axis(['a', 'b', 'c'], 'letters')
-        >>> letters.union(['b', 'c'])
-        Axis(['a', 'b', 'c'], 'letters')
+        >>> a = Axis('a=a0..a2')
+        >>> a.union(Axis('a=a1..a3'))
+        Axis(['a0', 'a1', 'a2', 'a3'], 'a')
+        >>> a.union('a1..a3')
+        Axis(['a0', 'a1', 'a2', 'a3'], 'a')
+        >>> a.union(['a1', 'a2', 'a3'])
+        Axis(['a0', 'a1', 'a2', 'a3'], 'a')
         """
+        if isinstance(other, basestring):
+            # TODO : remove [other] if ... when FuturWarning raised in Axis.init will be removed
+            other = [other] if '..' not in other and ',' not in other else Axis(other)
```

Comment:
use `_to_ticks(other, parse_single_int=True) if '..' in other or ',' in other else [other]` instead to avoid creating an Axis just to throw it away two lines below?